src/lib/generate-site-name.ts

```diff
@@ -36,14 +35,30 @@ export function generateSiteName( usedSiteNames: string[] ): string {
 }
 
 export const sanitizeFolderName = ( filename: string ) => {
+	const LATIN = 'a-z';
+	const CYRILLIC = 'а-яё';
+	const ARABIC = '\\u0600-\\u06FF';
+	const HEBREW = '\\u0590-\\u05FF';
+	const CHINESE = '\\u4e00-\\u9fa5';
+	const JAPANESE_HIRAGANA = '\\u3040-\\u309F';
+	const JAPANESE_KATAKANA = '\\u30A0-\\u30FF';
+	const KOREAN_HANGUL = '\\uAC00-\\uD7AF';
```

Comment:
The sanitization doesn't work for a Korean as NFKD normalization removes all Korean characters, producing an empty path.

Any smart way on how to solve that?

Reply:
Potential ways:
- Don't normalize string if a user uses Studio in Korean. It's an easy way, but edge cases won't work well e.g. user who uses Studio in Korean and uses Polish characters in the site name.
- replace Korean characters with placeholders before normalization and replace them back after. It would work in all cases, but seems overly complex.

Reply:
Yeah, I think we can skip normalization if Studio is in Korean.

Reply:
It seems I debugged it incorrectly and came to the wrong conclusion.

Korean syllables (Hangul) are decomposed by NKFD normalization into letters (Jamo). The initial regex included only syllables, so all letters were removed. I added Korean letters to the allowed list, which now works for all cases.

Reply:
Thanks for the fix, it looks good for me now 👍 

> I added Korean letters to the allowed list, which now works for all cases.

Nice work!